workers/generic-worker/schemas/insecure_posix.yml

```diff
@@ -177,6 +177,15 @@ oneOf:
               encoding to all the files contained in the directory.
 
               Since: generic-worker 16.2.0
+          optional:
+            title: Optional artifact
+            type: boolean
+            default: false
+            description: |-
+              If `true`, the artifact is optional. If the optional artifact doesn't exist and
```

Comment:
Technically, I think this should be phrased more like "if the file or directory doesn't exist, the artifact won't be created". The artifact isn't the file/dir itself, it is something that is created from the file/dir (and e.g. has its own attributes).